bookwyrm/models/book.py

```diff
@@ -201,14 +201,13 @@ def edition_info(self):
     @property
     def alt_text(self):
         """image alt test"""
-        text = self.title
-        if self.edition_info:
-            text += f" ({self.edition_info})"
-        return text
+        author = f"{name}: " if (name := self.author_text) else ""
+        edition = f" ({info})" if (info := self.edition_info) else ""
+        return f"{author}{self.title}{edition}"
```

Comment:
I think I see what you mean, but in that case... isn't this string a bit useless in translation context, since it's fully formatted already? (with colones etc.—this format being what might need changes in translation).

What I mean is: to make this translatable, shouldn't we produce three translation strings instead?:

- `{author}: {title}`
- `{title} ({edition)`
- `{author}: {title} ({edition})`

(At this point could also be `{title} by {author} ({edition})`, but that's a separate matter.)

Reply:
Ah I see what you mean, since it's formatted in previous variables, translating the final string would be basically meaningless. It would be best from a translation standpoint to enumerate fully formed strings (even though that's annoying from a code standpoint), but I don't think that's a blocker to merging, since it already isn't translated